Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -66,7 +66,6 @@ public partial class GDAXBrokerage : BaseWebsocketsBrokerage, IDataQueueHandler
         public GDAXBrokerage(string wssUrl, IWebSocket websocket, IRestClient restClient, string apiKey, string apiSecret, string passPhrase, IAlgorithm algorithm)
             : base(wssUrl, websocket, restClient, apiKey, apiSecret, Market.GDAX, "GDAX")
         {
-            throw new Exception("GDAX currently under maintenance and will be back online Monday 23rd October 2017");
```

Comment:
I removed this in https://github.com/QuantConnect/Lean/pull/1237/commits/410348b9a89a849144e1b6c98ebe6e2c07532281
strange it got back in.